File: C/level1/5 - Tests et conditions/5 - Tarif du bateau.c
//coded by cybereagle2001
#include <stdio.h>

int main()
{
   int age;
   scanf("%d", &age);
   
   if (age < 21)
   {
      printf("Tarif réduit");
   }
   else
   {
      printf("Tarif plein");
   }
}

File: java/level 1/5- Tests et conditions/5- Tarif du bateau.java
/** coded by cybereagle2001*/

import algorea.Scanner;

class Main{
   static Scanner input= new Scanner(System.in);
   public static void main(String[] args){
      int age= input.nextInt();
      if (age <21 ){
         System.out.println("Tarif réduit");
      }
      else{
         System.out.println("Tarif plein");
      }
   }
}
